### PR TITLE
KG - Create New Task Modal Bug Fix

### PR DESCRIPTION
**Pull request by kyle-glick**

### CODE REVIEW

File: app/assets/javascripts/tasks.js.coffee [hunk 1]
```diff
@@ -2,6 +2,11 @@ $ ->
 
   $('[data-toggle="tooltip"]').tooltip()
 
+  $(document).on 'click', '.new-task', ->
+    $.ajax
+      type: 'GET'
+      url: "/tasks/new"
```
Comment (goleksiak) [inline app/assets/javascripts/tasks.js.coffee]: Should add :format to url

**Comment by kyle-glick** [comment]
Will update the format on the URL

**Comment by kyle-glick** [comment]
This is ready for review

**Comment by goleksiak** [comment]
Looks ready to merge

**Comment by ctt3** [comment]
Looks good to me

**Pull request merged**